lib: bluetooth: enforce app PHY mask and support multi-PHY config (#788)
nrfconnect/sdk-nrf-bm: doc/nrf-bm/libraries/bluetooth/ble_conn_params.rst

The radio PHY mode defaults to 1 MB per second at the start of a connection.
This can be changed by initiating a GAP radio PHY mode update procedure.

> Choose either automatic PHY selection, or disable auto and enable one or more allowed PHYs:

Reviewer: ```suggestion
Choose either automatic PHY selection, or disable it and enable one or more allowed PHYs:
```

* :kconfig:option:`CONFIG_BLE_CONN_PARAMS_PHY_AUTO` - The SoftDevice will automatically select the PHY mode.

> * :kconfig:option:`CONFIG_BLE_CONN_PARAMS_PHY_1MBPS` - Allow 1 MB per second.

Reviewer: ```suggestion
* :kconfig:option:`CONFIG_BLE_CONN_PARAMS_PHY_1MBPS` - Allow the 1 Mbps PHY (1M PHY).
```

> * :kconfig:option:`CONFIG_BLE_CONN_PARAMS_PHY_2MBPS` - Allow 2 MB per second.

Reviewer: ```suggestion
* :kconfig:option:`CONFIG_BLE_CONN_PARAMS_PHY_2MBPS` - Allow the 2 Mbps PHY (2M PHY) for higher throughput.
```
